recipes/quill/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+from conans import ConanFile, CMake, tools
+from conans.model.version import Version
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class QuillConan(ConanFile):
+    name = "quill"
+    description = "C++14 Asynchronous Low Latency Logging Library"
+    homepage = "https://github.com/odygrd/quill/"
+    topics = ("conan", "quill", "logging", "log")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fmt_external": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fmt_external": False, "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.options.fmt_external:
+            self.requires("fmt/6.1.2")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        if self.options.shared:
+            self._cmake.definitions["BUILD_SHARED_LIBS"] = "ON"
+            self._cmake.definitions["BUILD_STATIC_LIBS"] = "OFF"
+        else:
+            self._cmake.definitions["BUILD_SHARED_LIBS"] = "OFF"
+            self._cmake.definitions["BUILD_STATIC_LIBS"] = "ON"
+
+        self._cmake.definitions["QUILL_FMT_EXTERNAL"] = "ON" if self.options.fmt_external else "OFF"
```

Comment:
```suggestion
        self._cmake.definitions["BUILD_STATIC_LIBS"] = not self.options.shared
        self._cmake.definitions["QUILL_FMT_EXTERNAL"] = self.options.fmt_external
```

- Conan converts boolean automatically to CMake defines
- BUILD_SHARED_LIBS is automatically configured by CMake helper: https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#definitions